### PR TITLE
add valid URIs in output for 0.0.0.0

### PR DESCRIPTION
**Pull request by bNobo**
Leave 0.0.0.0 to listen on every local interfaces but add localhost and 127.0.0.1 in output to ease usage.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -60,9 +60,16 @@ app.listen(
     HOST,
     () =>
         {
-            console.log(
-                `Http-echo is now running on http://${HOST}:${PORT}`
-            );
+            if (HOST === "0.0.0.0") {
+                console.log('Http-echo is now running on :');
+                console.log(`http://localhost:${PORT}`);
+                console.log(`http://127.0.0.1:${PORT}`);
```
Comment (r-cass) [inline index.js:66]: Must notice either for 0.0.0.0

Comment (bNobo) [reply index.js:66]: No both are valid. That's the point.

Comment (r-cass) [reply index.js:66]: I suggest to list all avalaible network addresses, like: https://stackoverflow.com/a/8440736

Comment (bNobo) [reply index.js:66]: http://0.0.0.0:${PORT} does not work in the browser, so I don't see the betterment of showing it in the output. As you said to me the server is not listening at this specific address "0.0.0.0", it is a convention to tell Express to bind every local interfaces.

Pleaaaaase don't be so fussy ! All I want is to have 'http://127.0.0.1' and/or 'http://localhost' in the output so I don't have to type it myself again and again. If I don't have it soon, I will abandon http-echo and use another solution or stick on my fork.

**Pull request merged**